### ISSUE TITLE
Update ESLint to use .eslintrc.js

### ISSUE THREAD

**Issue by rvarbanov** [MiguelCastillo/Brackets-InteractiveLinter#177]
On the [ESLint](http://eslint.org/docs/user-guide/configuring#configuration-file-formats) site they say that `.eslintrc` is deprecated and that we should use one of the following in that order:

1. `.eslintrc.js`
2. `.eslintrc.yaml`
3. `.eslintrc.yml`
4. `.eslintrc.json`
5. .`eslintrc` (deprecated)
6. `package.json`

I could just update the eslint's settings.json to one of those, but that did not seem to work well for me. Is there a way to update the Interactive Linter to look for one of those files?

**Comment by MiguelCastillo** [comment]
Ah ok. Yeah, it is about time we updated eslint...  Would you be interested in submitting a PR to update ESLint and the config?

**Comment by rvarbanov** [comment]
I could, but it is not working. That is why I submit an issue and not just a PR 😄 

I guess I could submit a PR with the eslint.js only... Is that what you are thinking of?

When I updated the settings.json file to look for `.eslintrc.js` I started getting errors, but it worked with `.eslintrc`

**Comment by MiguelCastillo** [comment]
i would submit a PR just to update ESLint...  We can tackle the other problem separately.

But why is your `.eslintrc` a `.eslintrc.js` and not a `.eslintrc.json`?

**Comment by rvarbanov** [comment]
ESLint supposed to look for `.eslintrc.js` first and also I like single quotes better 😃

**Comment by MiguelCastillo** [comment]
Currently, i only support JSON.  Which means that you really cant use single quotes; it has to be valid JSON :/

**Comment by MiguelCastillo** [comment]
Also, I know this isn't obvious at all.  But I run an independent version of ESLint in a worker thread.  So ESLint does not really load the configuration files itself when you are working with Interactive Linter.  This is one of the reasons I was saying in the other thread why I need to rethink this whole thing. :)

**Comment by rvarbanov** [comment]
Right, and that is why I created the issue. It would be nice if there is a way to allow for an array of settingsFiles. I do not know if that is achievable.

Again, I not know much about how the brackets extensions work so I am only suggesting how it could be improved but have no idea how to do it :)

**Comment by MiguelCastillo** [comment]
right. I completely missed that's what you were saying. sorry about that!

fair enough. it is a great suggestion.  moving to running the native eslint will allow us to specify eslint configurations in whichever way you want.  That's ultimately what we probably should do.
